UI/Selenium/appsettings.json

```diff
@@ -1,10 +1,42 @@
 {
-
+  "NLog": {
```

Comment:
See my comment in the nlog settings file.  All configs should be here unless you must use nlog.settings - if so, move the settings in here to nlog.

Reply:
I have removed NLog.Settings. We now use appsettings.json to configurate NLog